pkgs/dart_services/lib/src/api_model.dart

```diff
@@ -0,0 +1,204 @@
+// Copyright (c) 2023, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:json_annotation/json_annotation.dart';
+
+part 'api_model.g.dart';
+
+@JsonSerializable()
+class SourceRequest {
```

Comment:
This might be a good opportunity to have a shared package between the client and server, now that both packages live in the same repository, rather than maintaining a copy of this file in dart_pad and sketch_pad

Reply:
👍